Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:0.12.5
+RUN apt-get update && apt-get install -y --no-install-recommends python3-minimal python3-virtualenv python3-pip python3-dev python3-pil
+RUN python3 -m virtualenv -p /usr/bin/python3 --system-site-packages /var/www/shoop/venv
```

Comment:
Let's use virtualenv then. Dunno what's the base image of node:0.12.5, but if it's Ubuntu, then probably the reason is this bug: https://bugs.launchpad.net/ubuntu/+source/python3.4/+bug/1290847